src/main/java/org/spongepowered/common/mixin/core/entity/passive/MixinEntityWolf.java

```diff
@@ -90,4 +95,19 @@ public void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators)
         manipulators.add(get(SittingData.class).get());
         manipulators.add(new SpongeAggressiveData(this.isAngry()));
     }
+
+    @Inject(method = "processInteract", locals = LocalCapture.CAPTURE_FAILHARD,
+                at = @At(value = "FIELD", ordinal = 0, target = "Lnet/minecraft/entity/player/PlayerCapabilities;isCreativeMode:Z"))
```

Comment:
If you're using an ordinal, I'd rather you used a Slice, like from [here](https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java#L1010-L1028).